Simbody/Visualizer/src/VisualizerGeometry.h

```diff
@@ -51,6 +51,9 @@ class VisualizerGeometry : public DecorativeGeometryImplementation {
     void implementTextGeometry(const DecorativeText& geom);
     void implementMeshGeometry(const DecorativeMesh& geom);
     void implementMeshFileGeometry(const DecorativeMeshFile& geom) {}; // Not handled yet by this Visualizer
+    void implementArrowGeometry(const DecorativeArrow& geom) {}; // Not handled yet by this Visualizer
+    void implementTorusGeometry(const DecorativeTorus& geom) {}; // Not handled yet by this Visualizer
+    void implementConeGeometry(const DecorativeCone& geom) {}; // Not handled yet by this Visualizer
```

Comment:
We should file an issue on this when this code gets checked in. Maybe some outside contributor will get excited about it!